app/controllers/organizations/staff/external_form_upload_controller.rb

```diff
@@ -21,6 +22,11 @@ def create
           format.turbo_stream
         end
       end
+
+      def validate_attachment
```

Comment:
Can we leave this as two separate `before_action` calls :)
It makes it easier to read them both at the top, instead of scrolling down to read both.